build.fsx

```diff
@@ -54,7 +54,7 @@ Target "Build" (fun _ ->
     let additionalArgs = if versionSuffix.Length > 0 then [sprintf "/p:VersionSuffix=%s" versionSuffix] else []  
 
     if (isWindows) then
-        let projects = !! "./**/*.csproj"
+        let projects = !! "./**/*.csproj"// ++ "./**/*.fsproj"
```

Comment:
F# projects are still in this repo, but they are removed from solution and conditionally removed from tests. For some reason I was unable to make dotnet cli work for them.

Reply:
Just use the VS 2017 15.3 + netcoresdk 2.0. FSharp should work